src/main/java/core/basesyntax/Quadrangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Quadrangle extends Figure implements Area {
+    private final int width;
+    private final int height;
+
+    public Quadrangle(String color, int width, int height) {
+        super(color);
+        this.width = width;
+        this.height = height;
+    }
+
+    @Override
+    public double area() {
+        return width * height;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
Don't put all behavior into a single interface if the methods are conceptually different from each other. The `Area` interface should not contain the `draw()` method. You should have separate interfaces for different responsibilities.